vector/src/main/java/im/vector/app/features/widgets/WidgetPostAPIHandler.kt

```diff
@@ -63,19 +63,19 @@ class WidgetPostAPIHandler @AssistedInject constructor(@Assisted private val roo
     override fun handleWidgetRequest(mediator: WidgetPostAPIMediator, eventData: JsonDict): Boolean {
         return when (eventData["action"] as String?) {
             "integration_manager_open" -> handleIntegrationManagerOpenAction(eventData).run { true }
-            "bot_options"              -> getBotOptions(mediator, eventData).run { true }
-            "can_send_event"           -> canSendEvent(mediator, eventData).run { true }
-            "close_scalar"             -> handleCloseScalar().run { true }
-            "get_membership_count"     -> getMembershipCount(mediator, eventData).run { true }
-            "get_widgets"              -> getWidgets(mediator, eventData).run { true }
-            "invite"                   -> inviteUser(mediator, eventData).run { true }
-            "join_rules_state"         -> getJoinRules(mediator, eventData).run { true }
-            "membership_state"         -> getMembershipState(mediator, eventData).run { true }
-            "set_bot_options"          -> setBotOptions(mediator, eventData).run { true }
-            "set_bot_power"            -> setBotPower(mediator, eventData).run { true }
-            "set_plumbing_state"       -> setPlumbingState(mediator, eventData).run { true }
-            "set_widget"               -> setWidget(mediator, eventData).run { true }
-            "m.sticker"                -> pickStickerData(mediator, eventData).run { true }
+            "bot_options" -> getBotOptions(mediator, eventData).run { true }
+            "can_send_event" -> canSendEvent(mediator, eventData).run { true }
+            "close_scalar" -> handleCloseScalar().run { true }
+            "get_membership_count" -> getMembershipCount(mediator, eventData).run { true }
+            "get_widgets" -> getWidgets(mediator, eventData).run { true }
+            "invite" -> inviteUser(mediator, eventData).run { true }
+            "join_rules_state" -> getJoinRules(mediator, eventData).run { true }
+            "membership_state" -> getMembershipState(mediator, eventData).run { true }
+            "set_bot_options" -> setBotOptions(mediator, eventData).run { true }
+            "set_bot_power" -> setBotPower(mediator, eventData).run { true }
+            "set_plumbing_state" -> setPlumbingState(mediator, eventData).run { true }
+            "set_widget" -> setWidget(mediator, eventData).run { true }
+            "m.sticker" -> pickStickerData(mediator, eventData).run { true }
```

Comment:
Could you revert the formatting changes please? There is a problem in the IDE about that, and we have not decided if we want to lose the nice alignment of arrows. The trick is to remove the latest `}` of the file and type it again. The formatting will be restored...

Reply:
Haha rip. Sure!

Reply:
WOW! You weren't kidding about the trick. :joy: